lib/index.ts

```diff
@@ -12,11 +12,34 @@ import * as transport from './transport';
 import * as v8 from './v8';
 import * as zone from './zone';
 
-export { log, memory, metric, runtime, store, sync, transport, v8, zone };
+const napa = { log, memory, metric, runtime, store, sync, transport, v8, zone };
+module.exports = napa;
 
 // Add execute proxy to global context.
 import { call } from './zone/function-call';
 (<any>(global))["__napa_zone_call__"] = call;
 
 // Export 'napa' in global for all isolates that require napajs.
-(<any>(global))["napa"] = exports;
\ No newline at end of file
+(<any>(global))["napa"] = napa;
+
+// Prepare global store __napa_internal
+declare var __in_napa : any;
+const store_internal = (typeof __in_napa === 'undefined')
+                           ? napa.store.create('__napa_internal')
+                           : napa.store.get('__napa_internal');
```

Comment:
napa.store.create will be called more than once when require('napajs') is called for more than once in node by this logic.